### DISCUSSION TITLE
Changing the cover art name

### DISCUSSION THREAD

**Discussion by cleanyoursockets** [enzo1982/freac#427] [Help & Support]
Hello,
I like to know if it is possible to change the name of the cover art? I like it to be Folder.jpg instead of Front.jpg

Thanks

**Comment by enzo1982** [comment]
You can set the filename pattern for cover art files in the _Cover art_ tab of the _Tags_ page in the configuration dialog.

The default is to use the `<type>` placeholder which will resolve to _front_, _back_, _leaflet_, _disc_ etc. One caveat when setting this to e.g. `folder`, however, is that multiple album art files will overwrite each other when the `<type>` placeholder is not used in the pattern.

I plan to add additional configuration options for saving cover art files in a future update to solve these shortcomings.

**Comment by cleanyoursockets** [comment]
Thanks for the info, looking forward to the update, thank you.
